packages/bridge/src/nomic/index.ts

```
@@ -1,30 +1,378 @@
+    }
+    const nomicBridgeAsset: BridgeAsset = {
+      address: nomicBtc.coinMinimalDenom,
+      decimals: nomicBtc.decimals,
+      denom: nomicBtc.symbol,
+      coinGeckoId: nomicBtc.coingeckoId,
+    };
+    const ibcProvider = new IbcBridgeProvider(this.ctx);
+
+    const transactionDataParams: GetBridgeQuoteParams = {
+      ...params,
+      fromAmount: !!swapRoute
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)